lib/shims/eq/R2015b/isfolder.m

```diff
@@ -0,0 +1,9 @@
+function result = isfolder(folderName)
+
+% ISFOLDER Introduced in R2017b
+% ISFOLDER Determine if input is folder
```

Comment:
These two lines should be the other way around: the H1 line (first line of helptext) should have a single-line summary of the function's behavior; notes about when it was introduced and other stuff can come later.

Blank commet line between the H1 line and other helptext, please.

Reply:
Those comment lines weren't intended for helptext.  I mistakenly believed that by leaving a blank line above those comments, they would be in the source but the helptext would be null.  I see now that is wrong.  So yes, I'll reorganize the comments so that they are consistent with typical matlab helptext.  But many, many helptexts do not have a blank comment line after H1.  Wordsmithing at that level -- like choosing where to do a paragraph break -- might be too much to ask of me :-/

Reply:
helptexts have been rewritten, and PR is updated

Reply:
New helptext is looking good! Don't worry about the in-comment extra blank line and stuff.